tests/test_wave_05.py

```diff
@@ -31,6 +31,7 @@ def test_items_have_condition_as_float():
     for item in items:
         assert item.condition == pytest.approx(3.5)
 
+
 def test_items_have_condition_descriptions_that_are_the_same_regardless_of_type():
     items = [
```

Comment:
See comment about the README.md, I don't think you meant to include this empty space as part of your PR.